### PR TITLE
Add a div on hosting page to indicate that user is interested in hosting

### PR DESCRIPTION
**Pull request by mehulkar**
DO NOT MERGE

### CODE REVIEW

File: app/views/static/hosting.html.erb [hunk 1]
```diff
@@ -64,6 +64,9 @@
     </p>
     <h3 class="static-subheader">Join us!</h3>
 
+    <div class="interested-in-hosting hidden">
+      <p>LOREM IPSUM MY BUTT</p>
```
Comment (mehulkar) [inline app/views/static/hosting.html.erb]: @ankitshah811 change the copy here

Comment (ankitshah811) [inline app/views/static/hosting.html.erb]: i like this copy though

**Comment by ankitshah811** [comment]
Sorry, I don't think I was supposed to commit https://github.com/TeaWithStrangers/tws-on-rails/commit/11688c61a1e46165b7c6645fe496c921d0d49fbf, but branched this off my local master, which had cusotmizable email edits on it

**Comment by ankitshah811** [comment]
@mehulkar just a heads up that this is ready to go aside from the mistake commit (unless im missing something)

**Comment by mehulkar** [comment]
kk, cool. reviewing/rebasing and will merge soon.

**Pull request merged**